examples/fork-join/fork.description

```diff
@@ -0,0 +1,5 @@
+// In a function execution, a fork-join statement splits the current execution to multiple workers, which do their work in
+// parallel, and joins the executions together to process their results.
+// The `fork` block allows you to start any number of parallel workers at once.
+// Workers are visible outside the fork block as futures. You may wait for the worker returns using `wait` expression.
```

Comment:
Shall we expand on "futures" here? Maybe we can add some introduction so that a new user can follow this easily.